backend/python_backend/training_routes.py

```
@@ -24,12 +25,17 @@
 
 @router.post("/api/training/start")
 @log_performance(operation="start_training")
-async def start_training(model_config: ModelConfig, background_tasks: BackgroundTasks):
+async def start_training(
+    model_config: ModelConfig,
+    current_user: str = Depends(get_current_active_user),
+    background_tasks: BackgroundTasks,
```

Pylint (4.0.1)

[error] Parsing failed: 'non-default argument follows default argument (backend.python_backend.training_routes, line 31)'

(E0001)